helm-chart/scc-admin-frontend/templates/deploy-secret-serviceaccount.yaml

```
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "scc-admin-frontend.serviceAccountName" . }}-deploy-secret
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)